root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
+  }
+
+  return text;
+}
+
+const LanguageLink = ({$c, language}) => (
+  <a
+    href={
+      '/set-language/' + encodeURIComponent(language.name) +
+            '?' + returnToCurrentPage($c)
+    }
+  >
+    {languageName(language, false)}
+  </a>
+);
+
+type LanguageMenuProps = {
+  +$c: CatalystContextT,
+  +currentBCP47Language: string,
+  +serverLanguages: $ReadOnlyArray<ServerLanguageT>,
+};
+
+const LanguageMenu = ({
+  $c,
+  currentBCP47Language,
+  serverLanguages,
+}: LanguageMenuProps) => (
+  <div className="dropdown pb-2 ms-3 dropup">
+    <button
+      aria-expanded="false"
+      className="btn btn-outline-primary fs-5"
+      data-bs-toggle="dropdown"
+      id="language-dropdown"
+      type="button"
+    >
+      {languageName(
+        serverLanguages.find(x => x.name === currentBCP47Language),
+        true,
+      )}
+    </button>
+    <ul aria-labelledby="language-dropdown" className="dropdown-menu">
+      {serverLanguages.map(function (language, index) {
+        let inner = <LanguageLink $c={$c} language={language} />;
+
+        if (language.name === currentBCP47Language) {
+          inner = <strong>{inner}</strong>;
+        }
 
-const Footer = (): React.Element<'div'> => {
+        return (
+          <li
+            className="nav-item dropdown fs-5"
+            key={index}
+          >
+            {inner}
+          </li>
+        );
+      })}
+      <li className="nav-item dropdown fs-5">
+        <a href={'/set-language/unset?' + returnToCurrentPage($c)}>
+          {l('(reset language)')}
+        </a>
+      </li>
+      <li className="nav-item dropdown fs-5">
+        <a href="https://www.transifex.com/musicbrainz/musicbrainz/">
+          {l('Help Translate')}
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const Footer = (): React.Element<'section'> => {
   const $c = React.useContext(CatalystContext);
-  const stash = $c.stash;
+  const serverLanguages = $c.stash.server_languages;
+
   return (
-    <div id="footer">
-      <p className="left" id="footer-menu">
-        <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
-        <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
-        <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
-        </a>
-        <a className="internal" href="http://tickets.metabrainz.org/">{l('Bug Tracker')}</a>
-        <a className="internal" href="https://blog.metabrainz.org/">{l('Blog')}</a>
-        <a className="internal" href="https://twitter.com/MusicBrainz">{l('Twitter')}</a>
-
-        {DBDefs.BETA_REDIRECT_HOSTNAME ? (
-          <a
-            className="internal"
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
```

Comment:
As far as I can tell you've removed the beta link completely. Please bring it back? :) It could be by the language selector if you want, but you can pick a good place really.